easybuild/easyconfigs/c/CNVkit/CNVkit-0.9.6-foss-2019a-Python-3.7.2-R-3.6.0.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'PythonBundle'
+
+name = 'CNVkit'
+version = '0.9.6'
+versionsuffix = '-Python-%(pyver)s-R-%(rver)s'
+
+homepage = 'http://github.com/etal/cnvkit'
+description = """A command-line toolkit and Python library for detecting copy
+ number variants and alterations genome-wide from high-throughput sequencing."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('Biopython', '1.73'),
+    ('matplotlib', '3.0.3', '-Python-%(pyver)s'),
+    ('networkx', '2.3', '-Python-%(pyver)s'),
+    ('Pillow', '6.0.0'),
+    ('Pysam', '0.15.2'),
+    ('PyYAML', '5.1'),
+    ('R', '3.6.0'),
+    ('R-bundle-Bioconductor', '3.9', '-R-%(rver)s'),
+]
+
+use_pip = False
+
+exts_list = [
+    ('reportlab', '3.5.23', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/reportlab'],
+        'checksums': ['6c81ee26753fa09062d8404f6340eefb02849608b619e3843e0d17a7cda8798f'],
+    }),
+    ('pyfaidx', '0.5.5.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyfaidx'],
+        'checksums': ['9ac22bdc7b9c5d995d32eb9dc278af9ba970481636ec75c0d687d38c26446caa'],
+    }),
+    ('future', '0.17.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/future'],
+        'checksums': ['67045236dcfd6816dc439556d009594abf643e5eb48992e36beac09c2ca659b8'],
+    }),
+    ('pomegranate', '0.11.1', {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/jmschrei/pomegranate/archive'],
+        'checksums': ['2663db98fc64fac6d2c732d05f41b1a961f820e788b044f216fd3bf7b049646f'],
+    }),
+    (name, version, {
+        'installopts': "&& cd test && make && make test",
```

Comment:
these are not really install options? maybe try this instead:
```
'runtest': 'cd test && make && make test',
```

Reply:
@smoors That's probably because the tests required that `CNVkit` is installed first.

Via `runtest`, the tests are run before the actual installation.

It's probably a good idea to clarify that with a comment above this line in the easyconfig...

Reply:
I took the `installopts` from the previous easyconfig without much testing honestly. I made a few tests and that is indeed the case, the tests need CNVkit and all other extensions to be available.